test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/VisualTest.cs

```diff
@@ -12,10 +11,13 @@ public class VisualTest : IExample
 
         static private readonly string focusIndicatorImageUrl = DEMO_IMAGE_DIR + "i_focus_stroke_tile_2unit.9.webp";
         static private readonly string[] ImageUrlList = {
-            DEMO_IMAGE_DIR + "Dali/DaliDemo/application-icon-1.png",
-            DEMO_IMAGE_DIR + "Dali/DaliDemo/application-icon-6.png",
-            DEMO_IMAGE_DIR + "Dali/DaliDemo/Kid1.svg",
+            //DEMO_IMAGE_DIR + "Dali/DaliDemo/application-icon-1.png",
+            //DEMO_IMAGE_DIR + "AGIF/dog-anim.gif",
+            //DEMO_IMAGE_DIR + "Dali/DaliDemo/Kid1.svg",
```

Comment:
This comment is just to test this sample later?

Reply:
Those 3 icons has transparent background. So I'll upgrade this demo app later after prepare

For now, the best scene of Squircle feature is non-translucent image. So I choose 4 jpg images, and block 

But also, we might need to test transparent cases + animated cases. So just keep it as comments, so everyone can easily change the resources without found the resource path for this demo app :D